PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -1482,7 +1485,7 @@ def miniAODConfigurationPre(self, process, patMetModuleSequence, postfix):
             from PhysicsTools.PatAlgos.tools.metTools import addMETCollection
             addMETCollection(process,
                              labelName = "patCaloMet",
-                             metSource = "metrawCalo"+postfix if not self._parameters["Puppi"].value else "metrawCalo" 
+                             metSource = "metrawCalo" #+postfix if not self._parameters["Puppi"].value else "metrawCalo" 
```

Comment:
We found that it was not needed, and potentially dangerous, I am going to remove it